service/paxos/paxos_state.cc

```diff
@@ -60,8 +60,12 @@ future<prepare_response> paxos_state::prepare(storage_proxy& sp, db::system_keys
     lc.start();
 
     auto stats_updater = defer([&sp, schema, lc] () mutable {
-        auto& stats = sp.get_db().local().find_column_family(schema).get_stats();
-        stats.cas_prepare.mark(lc.stop().latency());
+        try {
```

Comment:
How about the following instead:
```c++
    if (auto table = sp.get_db().local().get_tables_metadata().get_table_if_exists(schema->id())) {
            auto& stats = table->get_stats();
            stats.cas_prepare.mark(lc.stop().latency());
    }
```

Reply:
I was looking for this method, but didn't find it.

Reply:
Done in v2.